app/services/user_service/send_card_locking_notification.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module UserService
+  class SendCardLockingNotification
+    def initialize(user:)
+      @user = user
+    end
+
+    def run
+      return unless Flipper.enabled?(:card_locking_2025_06_09, @user)
+
+      count = @user.transactions_missing_receipt(since: Date.parse("2025-06-09")).count
```

Comment:
```suggestion
      count = @user.transactions_missing_receipt(since: Date.new(2025, 6, 9)).count
```
instead of Date.parse